boards/rpi_5_scmi.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2024 EPAM Systems.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ /delete-node/ &sram0;
+
+#include <mem.h>
+
+/ {
+	#address-cells = <2>;
+	#size-cells = <1>;
+
+	sram0: memory@30000000 {
```

Comment:
I am wondering: should you describe the SCM/SCP  here as well? I.e. provide shared memory for SCMI, add scmi node, etc

Reply:
Our solution for Zephyr was not accepted. So current idea is to make Dom0 work without scmi - it should work the same way as before, without SCMI. So this patch just fixes the boot. The overlay can be extended later for SCMI.

Upstream PR for Zephyr SCMI is here https://github.com/zephyrproject-rtos/zephyr/pull/78293